src/psyclone/tests/psyir/transformations/loop_trans_test.py

```diff
@@ -183,3 +183,23 @@ def fake_validate(_1, _2, options=None):
                         trans.validate(target)
                 assert "validate test exception" in str(err.value), \
                     f"{name}.validate() does not call LoopTrans.validate()"
+
+
+def test_loop_trans_base_apply(fortran_reader):
+    '''Test the new base apply method for loop_trans. This now exists as we
+    need to specify the new options.'''
+    class fake_class(LoopTrans):
+        pass
+
+    psyir_test = fortran_reader.psyir_from_source('''
+       subroutine test()
+           integer :: i
+           integer, target :: a
+           integer, pointer :: b
+           do i = 1, 10
+               b => a
+           enddo
+       end subroutine test
+    ''')
+    loop = psyir_test.walk(Loop)[0]
+    fake_class().apply(loop)
```

Comment:
I guess this should do nothing as it's an empty method? Please could you check therefore that the transformation has left the code unchanged (with a comment explaining).

Reply:
Added.